components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/JSExecutionSupervisor.java

```diff
@@ -82,13 +116,12 @@ public void monitor(String identifier, String serviceProvider, String tenantDoma
      * @param identifier Monitoring task identifier.
      * @return Total elapsed time of adaptive auth execution in the current thread.
      */
-    public long completed(String identifier) {
+    public JSExecutionMonitorResult completed(String identifier) {
```

Comment:
Don't we want to stop monitoring for memory?

Something like
`memoryCounter.setThreadAllocatedMemoryEnabled(false);`

Reply:
Changed